jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryEndpointCaller.java

```diff
@@ -46,8 +46,8 @@
 class RegistryEndpointCaller<T> {
 
   /**
-   * <a
-   * href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/308">https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/308</a>.
+   * <a href=
```

Comment:
This is probably a change due to the previous formatter that didn't work. Please restore it to the original state.

Reply:
Done